weather_mv/loader_pipeline/util.py

```diff
@@ -85,13 +85,15 @@ def to_json_serializable_type(value: t.Any) -> t.Any:
     # Note: The order of processing is significant.
     logger.debug('Serializing to JSON')
 
-    if pd.isna(value) or value is None:
+    # pd.isna() returns ndarray if input is not scalar therefore checking if not ndarray.
+    if type(value) != np.ndarray and (pd.isna(value) or value is None):
         return None
     elif np.issubdtype(type(value), np.floating):
         return float(value)
     elif type(value) == np.ndarray:
         # Will return a scaler if array is of size 1, else will return a list.
-        return value.tolist()
+        # Replace all NaNs, NaTs with None.
```

Comment:
As mentioned above: can we replace all `type(x) == klass` with `isinstance(x, klass)`?

Reply:
Updated in https://github.com/google/weather-tools/pull/327/commits/dac23a047e2357b4f0dce9fb0b17969ae7f972a5.